modules/openy_features/openy_prgf/openy_prgf.install

```diff
@@ -103,3 +103,20 @@ function openy_prgf_update_8008() {
     );
   }
 }
+
+/**
+ * Update entity_clone setting to always mark paragraph entities to be cloned.
+ */
+function openy_prgf_update_8009() {
+  $config_importer = \Drupal::service('openy_upgrade_tool.param_updater');
```

Comment:
> Can we move `entity_clone.settings.yml` to optional configs and remove this dependency?

If this item fixed - let's check in the hook update that  `entity_clone` module enabled and import config only in this case.